concrete-test/src/synthesizing.rs

```diff
@@ -0,0 +1,96 @@
+//! A module to synthesize actual entities from prototypical entities.
+//!
+//! This module allows to convert back and forth, prototypical entities to the actual entity types
```

Comment:
`allows to convert back and forth between prototypical entities and the actual entity types`